obs-studio-server/source/nodeobs_display.cpp

```diff
@@ -305,6 +308,52 @@ OBS::Display::Display()
 	*v.color = 0xFFFFFFFF;
 	m_boxTris->Update();
 
+	// Rotation handle line
+	m_rotHandleLine = std::make_unique<GS::VertexBuffer>(5);
+	m_rotHandleLine->Resize(5);
+	v = m_rotHandleLine->At(0);
+	vec3_set(v.position, 0.5f - 0.34f / HANDLE_RADIUS, 0.5f, 0);
```

Comment:
"0.5f - 0.34f / HANDLE_RADIUS"
compiler will optimize it anyway but moving to a const variable will make code cleaner

Reply:
You are right! I will fix it.

Reply:
Actually, it turned out it is more complicated. I cannot convert `0.5f - 0.34f` to a constant variable because `0.34f / HANDLE_RADIUS` goes first (division goes first). If I pre-calculate the numbers entirely and remove HANDLE_RADIUS at all, then I have to do the same in `OBS::Display::DrawRotationHandle` because it depends on `m_rotHandleLine` and `HANDLE_RADIUS`.  Then, in some time, it will be difficult to understand why we use the numbers. OR do you mean something else?

Reply:
if it will complicate code then it is good to keep as is.